### PR TITLE
Displays all commits in a build changeset

### LINKED ISSUE #108: Show properly when multiple commits are in a bundle in build2
The build1 page nicely shows all the commit corresponding to a CI run. But build2 doesn't.
It would be helpful to at least get a visual marker on build2 indicating that this is a bundle of commits (so people should check build1 for other commits) or even better have the same multi-line visualization to know what are the other commits.

### PR DESCRIPTION
**Pull request by fiferdc**
This fixes #108 by exposing the commits in a changset when viewing build2.

Build1 commits:
Screen Shot 2021-09-10 at 9 59 41 AM

Build2 without change:
Screen Shot 2021-09-10 at 9 58 46 AM

Build2 with change:
Screen Shot 2021-09-10 at 9 55 57 AM

### CODE REVIEW

File: src/GitHubStatusDisplay.js [hunk 1]
```diff
@@ -813,7 +818,9 @@ export default class BuildHistoryDisplay extends Component {
       }
 
       function renderPullRequestNumber(comment) {
-        let m = comment.match(/\(#(\d+)\)/);
+        let m = comment.match(/\(#(\d+)\)/) || comment.match(
+          /https:\/\/github.com\/pytorch\/pytorch\/pull\/(\d+)/
+        );
```
Comment (malfet) [inline src/GitHubStatusDisplay.js:823]: Can you please explain how this change is relevant to the goal of displaying more PRs?

**Review by malfet — CHANGES_REQUESTED** [review]
Thank you for working on this issue.
Please run `yarn prettier --write src` and resubmit your PR.
Also, judging from the preview, after your PR commit description and pull request numbers are no longer rendered
image

Last but not the least, I don't think using Jenkins data is the right way to present those missing commits, as all those PRs are present in [index.json](https://s3.amazonaws.com/ossci-job-status/master/index.json) aren't they?)

File: src/GitHubStatusDisplay.js (hunk 1 above)
Comment (fiferdc) [reply src/GitHubStatusDisplay.js]: I am using the associated function to generate some Fragment.  I noticed that the code Fragments following each match were identical and combined the match functions to collapse the duplicated Fragment generation.

**Comment by fiferdc** [comment]
> Please run `yarn prettier --write src` and resubmit your PR.

I saw that once the test failed.  I was considering adding that to `package.json` to ease contributions, but that seemed like too much scope creep.

> Also, judging from the preview, after your PR commit description and pull request numbers are no longer rendered

They are still rendered.  The screen grab cut them off.

> Last but not the least, I don't think using Jenkins data is the right way to present those missing commits, as all those PRs are present in [index.json](https://s3.amazonaws.com/ossci-job-status/master/index.json) aren't they?)

While the commits will be present in that file, the changesets are not.  I had to rely on the Jenkins data to group the commits together and present a similar view to build1.  Looking at the source generating build1, the grouping there also stems from the Jenkins data.

**Comment by malfet** [comment]
@fiferdc I'm looking at https://deploy-preview-113--pytorch-ci-hud.netlify.app/build2/pytorch-master and I could not see them renderred, here is the screenshot using inspector:
image

Also, Jenkins system is deprecated and we are trying to get rid of it, GitHub Actions is the system that currently drives majority of GitHub CI and therefore should be used as source of truth.

Commenting out following 3 lines achieves the same goal:
https://github.com/pytorch/pytorch-ci-hud/commit/d2bc78f5d643834279775c25a3ed1e0652a2e3b6

I would greatly appreciate if you can adapt your PR to add toggle to the HUD that enables/disables this feature based on users preference

**Pull request closed without merging**